lab5.cpp

```diff
@@ -0,0 +1,66 @@
+﻿#include <iostream>
+// #include <string>
+
+using namespace std;
+
+class Fighter
+{
+public:
+    string name;
+    int health;
+    int damagePerAttack;
+
+    string getName()
+    {
+        return name;
+    }
+    int getHealth()
+    {
+        return health;
+    }
+    int getDamagePerAttack()
+    {
+        return damagePerAttack;
+    }
+
+    Fighter(string name, int health, int damagePerAttack)
+
+    {
+        this->name = name;
+        this->health = health;
+        this->damagePerAttack = damagePerAttack;
+    }
+};
+
+class Fight
+{
+public:
+    void declareWinner(Fighter fighter1, Fighter fighter2)
```

Comment:
pls when write functions open the brace (this -> { ) in same line

Reply:
I apologize again, but do you understand exactly how the code is written? If you write through (this->{) then you will need to change most of the conditions